<launch>

  <!--velo to laserscan-->
  <node pkg="pointcloud_to_laserscan" type="pointcloud_to_laserscan_node" name="pointcloud_to_laserscan_node_local" args="cloud_in:=/velodyne_points" respawn="false">
    <param name="range_min" value="1.2"/>
    <param name="range_max" value="100"/>
    <param name="min_height" value="-0.12"/>
    <param name="max_height" value="0.3"/>
    <param name="angle_increment" value="0.004360"/>
    <remap from="scan" to="scanlocal"/>
  </node>

</launch>
